symbols/arm9.yml

```
@@ -10482,6 +10482,16 @@ arm9:
         In particular, mission::status is set to mission_status::MISSION_STATUS_INVALID, mission::dungeon_id is set to -1, mission::floor is set to 0 and mission::reward_type is set to mission_reward_type::MISSION_REWARD_MONEY.
         
         r0: Pointer to the mission to clear
+    - name: GenerateMissionDetailsStruct
+      address:
+        EU: 0x20604CC
+      description: |-
+        Generates a mission_details struct, to be used for printing mission details such as the title, summary, and objective.
+        
+        r0: mission struct pointer
+        r1: [output] location of mission_details struct
+        r2: unknown
+        r3: unknown
```

The repo's standard for denoting unknown params is "?" I.e.

```suggestion
        r2: ?
        r3: ?
```